BossMod/Framework/Plugin.cs

```diff
@@ -93,6 +97,7 @@ public unsafe Plugin(IDalamudPluginInterface dalamud, ICommandManager commandMan
         _wndReplay = new(_ws, _rotationDB, replayDir);
         _wndRotation = new(_rotation, _amex, () => OpenConfigUI("Autorotation Presets"));
         _wndDebug = new(_ws, _rotation, _amex);
+        _qbWindow = new(_qb);
```

Comment:
Let's move this to debug window, I don't think there's anything typical user would want to see here?